com/course/zenkouv/src/main/java/lecture6/model/GitHubUser.java

```diff
@@ -0,0 +1,34 @@
+package lecture6.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GitHubUser {
+
+    private String login;
+    private int id;
+
+    public String getLogin() {
+        return login;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setLogin(String login) {
```

Comment:
unneeded setters

Reply:
remove getters. if i remove setters, this fields won't have any data.